SecretManagement.KeePass.Extension/Tests/Get-Secret.Tests.ps1

```diff
@@ -0,0 +1,185 @@
+Import-Module -Name 'Microsoft.PowerShell.SecretManagement'
```

Comment:
These two import lines should be in a BeforeAll{} block, otherwise it tries to import the modules during discovery which we don't want. See "Upgrading to Pester 5" documents for more info.

Reply:
Gonna need this fixed first, Tests need to not "run" anything during discovery phase.